docs/reference/resource-limits-fees.mdx

```diff
@@ -13,31 +13,13 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
 
 Soroban introduces smart contracts, marking the most transformative upgrade to the Stellar network to date. To maintain network performance, the ecosystem has decided to upgrade the network in a phased approach where network capacity is increased gradually over time. See [this blog post](https://stellar.org/blog/developers/the-stellar-network-s-phased-rollout-of-smart-contracts-the-road-to-mainnet) for more details and [Discord](https://discord.com/invite/zVYdY3ktTn) for current status.
 
-| Network Setting | Phase 0 | Phase 1 (current) |
```

Comment:
I didn't meant to remove the whole table, I meant we should just remove mentions of the phases and just have a single two column table (setting-value). Sorry for the confusion.
We don't need 2 separate tables for fees though, just one (with current value) would suffice.